tests/functional/package.json

```diff
@@ -20,7 +20,7 @@
     "cvg:summary": "nyc report --temp-dir=.nyc_output --reporter=text-summary --cwd=. --exclude-after-remap false"
   },
   "dependencies": {
-    "@heartexlabs/ls-test": "heartexlabs/ls-frontend-test#9ff949fc19abaf124477dfd41c6f230b00665ab3"
+    "@heartexlabs/ls-test": "heartexlabs/ls-frontend-test#2c9410fb930c7d920474fe6e624efee6261cec0c"
```

Comment:
don't forget to switch back to master version after merging changes in the testing library

Reply:
just did it a second ago